meson_options.txt

```diff
@@ -8,6 +8,11 @@ option('default-wallpaper',
        value: '/usr/share/backgrounds/elementaryos-default',
        description: 'Path to default wallpaper to use in Pantheon.')
 
+option('default-wallpaper-dark',
+       type: 'string',
+       value: '',
```

Comment:
Should there be a value here like default-wallpaper?

Reply:
I suppose we could make this the same value as the light wallpaper?

Reply:
That would be fine to me. I only ask because default implies there is *something* but I did approve as it isn't a huge deal if there isn't anything. I assume this wouldn't have an effect on using Pantheon on other distros?

Reply:
I’m not sure exactly what downstreams are shipping, but this basically stops us from switching to the GNOME default dark wallpaper automatically by default haha

Reply:
```suggestion
       value: '/usr/share/backgrounds/elementaryos-default',
```